javascript-challenges/infinite-scroll/devashmit-day38/index.html

```
@@ -0,0 +1,31 @@
+.lbc{position:absolute;top:1rem;right:1.5rem;background:none;border:none;color:#fff;font-size:1.8rem;cursor:pointer}
+</style></head>
+<body>
+<header><h1>📸 Infinite Gallery</h1><p>Scroll to load more</p></header>
+<main>
+  <div class="gallery" id="gallery"></div>
+  <div class="spinner" id="spinner"><div class="ring"></div></div>
+  <div class="sentinel" id="sentinel"></div>
+</main>
+<div class="lb" id="lb" role="dialog" aria-modal="true">
+  <button class="lbc" id="lbc" aria-label="Close">✕</button>
+  <img id="lbImg" src="" alt=""/><p id="lbCap"></p>
```

HTMLHint (1.9.2)

[error] The attribute [ src ] of the tag [ img ] must have a value.

(src-not-empty)